bkg/v2/BKG_v2.X.yaml

```diff
@@ -3302,6 +3325,29 @@ components:
             - `2 alphanumeric characters` for the sequence number of the voyage
             - `1 character` for the direction/haul (`N`orth, `E`ast, `W`est, `S`outh or `R`oundtrip).
           example: 2103N
+        routingReference:
+          type: string
+          pattern: ^\S(?:.*\S)?$
+          maxLength: 100
+          description: |
+            A reference to a predifned `route` specified in the **Commercial Schedules - Point to point**. When specifying this property - it is not needed to specify the following properties:
```

Comment:
Typo in "predefined". 

Also, it says that is not needed to specify other properties, what happens if they are specified, which takes prevalence? or MUST NOT be used at same time?

Reply:
Good question - this would be defined as a "complex validation" in the Conformance Tool. I suggest that if `routingReference` is used together with any of the properties listed - it is an error (a PENDING_UPDATE)

Reply:
Typo fixed :-)